docs/bots.md

```diff
@@ -55,6 +55,34 @@ Example of creating the bot user with the API.
 curl -X POST -H "content-type: application/json" -H "x-rems-api-key: 42" -H "x-rems-user-id: owner" http://localhost:3000/api/users/create --data '{"userid": "rejecter-bot", "name": "Rejecter Bot", "email": null}'
 ```
 
+## Expirer bot
+
+User id: `expirer-bot`
+
+The Expirer bot is used to remove applications and send notification email to
+application members about impending expiration. Expirations can be configured
+by application state for both application removal and sending reminder email.
+
+Unlike other bots, the Expirer bot is designed to be triggered by an external
+event, such as a periodically ran process. The bot is supplied an application
+and it returns appropriate command description, if applicable. Responsibility
+of the command execution is left to the caller. A scheduler is implemented in
+REMS, enabled by config option `:expiration` to periodically call the Expirer
+bot.
+
```

Comment:
Should we mention around here that, if the person does something, the count starts from zero and another expiration notification is potentially sent later?